app/assets/v2/js/pages/profile-trust.js

```diff
@@ -256,7 +268,7 @@ Vue.component('active-trust-manager', {
         // check for a passport and then its validity
         if (passport) {
           // check if the stamps are unique to this user...
-          const stampHashes = await apiCall(`/api/v2/profile/${document.contxt.github_handle}/passport/stamp/check`, {
+          const stampHashes = await apiCall(`/api/v2/profile/${document.contxt.github_handle}/passport/stamp/sscheckk`, {
```

Comment:
Why are we changing this? I don't see a corresponding change in the `urls.py`?

Reply:
Good catch, I committed this by mistake.